test_list_sort.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from list_sort import list_sort
+
+
+class listsortTest(unittest.TestCase):
+
+    def test_integer_input(self):
+        self.assertEqual(list_sort(5), 'Invalid Input')
+
+    def test_string_input(self):
+        self.assertEqual(list_sort('string'), 'Invalid Input')
+
+    def test_empty_list(self):
+        self.assertEqual(list_sort([]), {'evens': [], 'odds': [], 'chars': []})
+
+    def test_no_string(self):
+        self.assertEqual(
+            list_sort([4, 3, 2]),
+            {'evens': [2, 4], 'odds': [3], 'chars': []}
+            )
+
+    def test_no_even(self):
+        self.assertEqual(
+            list_sort([9, 3, 5, 1, 'd', 'a']),
```

Comment:
Did you run these tests? Did they pass? I would expect them not to pass because you are calling a `list_sort` method that takes a list argument and yet the `list_sort` method you defined does not take any arguments.